.github/workflows/test-elixir.yaml

```diff
@@ -41,12 +40,19 @@ jobs:
         uses: actions/checkout@v3
 
       # Step: Setup Elixir + Erlang image as the base.
-      - name: Set up Elixir
+      - name: Set up Elixir on Windows or Linux
+        if: "{{ runner.os != 'macOS' }}"
```

Comment:
@kasvith I made a change here; I think this should reduce the duplication, lmk if that makes sense

Reply:
Thanks max, makes sense